internal/storage/bucket_handle_test.go

```diff
@@ -467,3 +470,162 @@ func (t *BucketHandleTest) TestUpdateObjectMethodWithMissingObject() {
 
 	AssertTrue(errors.As(err, &notfound))
 }
+
+func (t *BucketHandleTest) TestComposeObjectMethodWithOneValidSrcObject() {
```

Comment:
just say withOneSrcObject, valid is not required.

Reply:
fixed